ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-stable-openshift-logging-6.3.yaml

```diff
@@ -63,6 +63,40 @@ tests:
     - ref: install-operators
     - chain: openshift-e2e-test-logging-qe
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: azure-ipi-amd-f999-logging-custom-tls-profile
+  capabilities:
+  - intranet
+  cron: 8 8 29 2 *
+  steps:
+    cluster_profile: azure-qe
```

Comment:
Please use profile azure-observability and  BASE_DOMAIN: observability.azure.devcluster.openshift.com

Reply:
Fixed